backend/daos/mongoConnection.js

```diff
@@ -1,19 +1,24 @@
+/* eslint-disable no-undef */
 const mongoose = require("mongoose");
-const Config = require("config");
 
 async function connect() {
-  const address = Config.get('application.mongo.address');
```

Comment:
I can't get the npm config library working with docker-compose either in CI env or running it locally.

So I have instead actually been loading these configs/credentials from a .env file under the backend folder...